Makefile

```diff
@@ -1,42 +1,53 @@
-VIRTUALENV ?= python3 -m venv
+VIRTUAL_ENV ?= $(shell pwd)/venv
 
 all:: compile
 
 clean::
 	rm -rf build/contracts.json .requirements-installed
 
-lint: install-requirements
-	flake8 tests py-deploy
-	black --check tests py-deploy
-	mypy --ignore-missing-imports tests py-deploy
+lint: setup-venv install-requirements
+	$(VIRTUAL_ENV)/bin/flake8 tests py-deploy
+	$(VIRTUAL_ENV)/bin/black --check tests py-deploy
+	$(VIRTUAL_ENV)/bin/mypy --ignore-missing-imports tests py-deploy
 
-test:: install
-	pytest tests
+test:: setup-venv install
+	$(VIRTUAL_ENV)/bin/pytest tests
 
-.requirements-installed: dev-requirements.txt
+.requirements-installed: setup-venv dev-requirements.txt
 	@echo "===> Installing requirements in your local virtualenv"
-	pip install -q -r dev-requirements.txt
+	$(VIRTUAL_ENV)/bin/pip install -q -r dev-requirements.txt
 	@echo "This file controls for make if the requirements in your virtual env are up to date" > $@
 
 install-requirements:: .requirements-installed
 
-compile:: install-requirements
+compile:: setup-venv install-requirements
 	@echo "==> Compiling contracts"
-	deploy-tools compile --optimize
+	$(VIRTUAL_ENV)/bin/deploy-tools compile --optimize
 	cp -p build/contracts.json py-bin/
+	cp -p build/contracts.json $(VIRTUAL_ENV)/trustlines-contracts/build/
```

Comment:
I dont think that this step is necessary. This was apparently done before, but not always, so I think the proper way is find out what was copying it before and why this is not copying always when needed.

Reply:
No, it was not done before.
The `contracts.json` in `$(VIRTUAL_ENV)/trustlines-contracts/build/` comes from `pip install $(PIP_DEPLOY_OPTIONS) ./py-deploy` since we find `"trustlines-contracts-bin>=1.1.1,<2.0.0"` in `contracts/py-deploy/setup.py /` afaik.
I do not think there was a copying.